storage/block_devices.go

```diff
@@ -214,6 +221,19 @@ func getAliasSuffix(file string) string {
 	return ""
 }
 
+func (bd *BlockDevice) FindAllChildren() []*BlockDevice {
```

Comment:
Instead of this use FindAllBlockDevices from block_devices_filters.go

Reply:
I did look at this. I feel the locations where you call this , you have a lot of filter logic and that would become part of inner function in my approach. I tried but I wasn't able to create a much simplified solution with benefits. So I think you can overlook this. The commit i pushed addresses another location where simplification had benefits.

Reply:
@karthikprabhuvinod When testing your commit, it was working fine for the LVM, but it removed media for all Safe and Destructive installations. I've since dropped that commit.